modern/src/ReloadPrompt.tsx

```diff
@@ -0,0 +1,56 @@
+import { Snackbar, Button } from '@mui/material';
+import React from 'react'
+import { useTranslation } from './common/components/LocalizationProvider';
+import { useAttributePreference } from './common/util/preferences';
+import { useRegisterSW } from 'virtual:pwa-register/react'
+
+// Based on https://vite-pwa-org.netlify.app/frameworks/react.html
+function ReloadPrompt() {
+  const t = useTranslation();
+
+  const serviceWorkerUpdateInterval = useAttributePreference('serviceWorkerUpdateInterval', 3600000);
+
+  const {
+    offlineReady: [offlineReady, setOfflineReady],
```

Comment:
Why do we need this?

Reply:
Sorry this remained from the original snippet, will remove it.